packages/manager/src/features/CloudPulse/Alerts/AlertsListing/AlertActionMenu.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+import { ActionMenu } from 'src/components/ActionMenu/ActionMenu';
+
+import type { AlertDefinitionType } from '@linode/api-v4';
+
+export interface ActionHandlers {
+  // These handlers will be enhanced based on the alert type and actions required
+  /*
+   * Callback for delete action
+   */
+  handleDelete: () => void;
+
+  /*
+   * Callback for show details action
+   */
+  handleDetails: () => void;
+}
+
+export interface AlertActionMenuProps {
+  /*
+   * Type of the alert
+   */
+  alertType?: AlertDefinitionType;
+  /*
+   * Handlers for alert actions like delete, show details etc.,
+   */
+  handlers?: ActionHandlers;
+}
+
+/*
+The handlers and alertType are made optional only temporarily, they will be enabled but they are dependent on another feature which will be part of next PR
+*/
```

Comment:
For future reference: Currently, this shows up like this:

![Screenshot 2024-12-03 at 1 46 34 PM](https://github.com/user-attachments/assets/c101c314-815f-4de6-9aa0-9532b75c7416)

If you format your comment like this:
```suggestion
/**
 * The handlers and alertType are made optional only temporarily, they will be enabled but they are dependent on another feature which will be part of next PR
 */
```

Code editors will be able to display the comment in a more readable way

![Screenshot 2024-12-03 at 1 46 05 PM](https://github.com/user-attachments/assets/a003fe09-9ccb-4599-994b-5e6348a955b3)

Reply:
This was mainly to let the reviewers and developers know that the current file is in a temporary stage. Will make the change and will follow that for future PRs